library/src/main/java/com/novoda/downloadmanager/domain/DownloadFile.java

```diff
@@ -12,15 +12,13 @@
     private final long totalSize;
     private final String localUri;
     private final FileStatus fileStatus;
-    private final String fileIdentifier;
```

Comment:
They are IDs. The local URIs must be unique otherwise you'll get into an unstable state when you try to download two different (or same) remote files to the same local path. Since the local URIs must be unique, they're sufficient as IDs.